shared/Microsoft.AspNetCore.BenchmarkRunner.Sources/AspNetCoreBenchmarkAttribute.cs

```diff
@@ -2,38 +2,70 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Linq;
-using System.Reflection;
-using BenchmarkDotNet.Attributes;
-using BenchmarkDotNet.Running;
+using System.Collections.Generic;
 using BenchmarkDotNet.Configs;
-using BenchmarkDotNet.Jobs;
-using BenchmarkDotNet.Toolchains.InProcess;
 
 namespace BenchmarkDotNet.Attributes
 {
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Assembly)]
     internal class AspNetCoreBenchmarkAttribute : Attribute, IConfigSource
     {
-        public static bool UseValidationConfig { get; set; }
-
-        public Type ConfigType { get; }
-        public Type ValidationConfigType { get; }
-
-        public AspNetCoreBenchmarkAttribute() : this(typeof(DefaultCoreConfig))
+        public AspNetCoreBenchmarkAttribute()
+            : this(typeof(DefaultCoreConfig))
         {
         }
 
-        public AspNetCoreBenchmarkAttribute(Type configType) : this(configType, typeof(DefaultCoreValidationConfig))
+        public AspNetCoreBenchmarkAttribute(Type configType)
+            : this(configType, typeof(DefaultCoreValidationConfig))
         {
         }
 
         public AspNetCoreBenchmarkAttribute(Type configType, Type validationConfigType)
```

Comment:
You can subclass this and replace things if you have more or different configurations you care about. I left in the existing overloads for back-compat with existing code.